content/blog/2021-06-18-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,241 @@
+
+![So many choices...](https://media.giphy.com/media/Uni2jYCihB3fG/giphy.gif)
+
+## LJ Miranda's Triad of order
+
+The MLOps tool landscape can be confusing to say the least.  
+[LJ Miranda](https://twitter.com/ljvmiranda921), in a well written
+[three-part series](https://ljvmiranda921.github.io/notebook/2021/05/10/navigating-the-mlops-landscape/)
+lays out a framework for making sense of this space. The list of tools is not
+exhaustive, but the framework and thought process for evaluating the tools is
+intriguing. Additionally he encourages thinking about the skillset of the
+members of your team within this framework to help you make decisions on the
+right tools. It's not just about the tools, it's about the people!
+
+As you can see DVC makes it into the "Trial" loop, but we think we will be be
+making it into the adoption region in relatively short order. 😉🚀
+
+![](/uploads/images/2021-06-18/LJMiranda.png) _Making sense of the MLOps
+Landscape_
+
+## Found in the MLOps Community
+
+You can find more comments from LJ Miranda and others in response to a
+[great question](https://mlops-community.slack.com/?redir=%2Farchives%2FC015J2Y9RLM%2Fp1622714574054300)
+from André Godinho in the [MLOps Community](https://mlops.community/) Slack (see
+below). If you're into MLOps and you're NOT a part of this Community, you should
+be. You can join their Slack
+[here](https://mlops-community.slack.com/join/shared_invite/zt-o96abp9z-sRYKWb96wGK9vdhUvbSrsQ#/shared-invite/email).
+
+> I have recently came across with DVC by listening to MLOps Coffee Sessions #6
+> with David Aponte and Elle O'Brien (Such an interesting talk! 💯). This tool
+> integrates smoothly with Git, tracks models & datasets, and also has an online
+> UI DVC Studio 🚀. Is there any use case of MLflow that DVC can't handle? I
+> find DVC to give more rise to creativity as it integrates really well with
+> Git. - André Godinho
+
+## Neda Sultova's Tutorial and Tool Rubric
+
+Drilling down to the next level, I give you
+[this tutorial](https://medium.com/geekculture/exploring-dvc-for-machine-learning-pipelines-in-research-part-1-3ebc2ca35a18)
+by [Neda Sultova](https://www.linkedin.com/in/neda-sultova-597a811a8/). Not only
+is it a great tutorial of DVC in and of itself, but Neda also defines a clear
+framework for the decision making process at
+[Helmholtz AI](https://www.helmholtz.ai/). Among the needs are reproducibility,
+workflow integration, exchangeable backend, framework agnostic, open source, and
+the ability of the solution to be tweaked to the team's needs.
+
+<external-link
+href="https://medium.com/geekculture/exploring-dvc-for-machine-learning-pipelines-in-research-part-1-3ebc2ca35a18"
+title="Exploring DVC for Machine Learning Pipelines in Research (Part 1)"
+description="The first of a multi-part series on the search and decision making process for MLOps tools at Helmholtz AI."
+link="https://medium.com"
+image="/uploads/images/2021-06-15/neda-sultova.png"/>
```

Comment:
Looks like all these 6-15 image paths are broken (should be 6-18)
![image](https://user-images.githubusercontent.com/1477535/122622808-819db900-d05f-11eb-8320-785c5e614889.png)

Reply:
p.s. @rogermparent the link checker doesn't cover image paths right? I may try to add that to https://github.com/iterative/link-check/pull/6.

Reply:
Hmm I guess we'd also need to check `link="(.*)"` but is this a good idea? Since `external-link` is a custom element (I think). Please see https://github.com/iterative/link-check/pull/6/commits/52961f9f0ddbe89f0ae5351ac2ab46d150125b6f